Code/Tools/ProjectManager/Source/CreateProjectCtrl.cpp

```diff
@@ -268,12 +268,12 @@ namespace O3DE::ProjectManager
                 PythonBindingsInterface::Get()->AddProject(projectInfo.m_path);
 
 #ifdef TEMPLATE_GEM_CONFIGURATION_ENABLED
```

Comment:
Is there a reason to get the `TEMPLATE_GEM_CONFIGURATION_ENABLED` define around.
It should always be defined at this point, which reduces the usefulness of it